README.md

```diff
@@ -52,13 +55,18 @@ Use a [Trello](https://trello.com) board as a CMS to create & manage a website.
 2. Get your `TRELLO_APP_KEY` from [trello.com/app-key](https://trello.com/app-key)
 3. Using your key, get your `TRELLO_TOKEN` from `https://trello.com/1/authorize?expiration=never&scope=read&response_type=token&name=Husky%20CMS&key=__YOUR_KEY_HERE__`
 4. Make a Trello board for your site's content
-   - Add 4 lists: `Draft`, `Pages`, `Blog`, `Projects` and get the ids for them
-   - You'll need to get the id's of your lists to pass them to Husky
+   - Add 5 lists: `Draft`, `Pages`, `Blog`, `Projects`, `Timeline` and get the ids for them
      - Open a card on the list you want to get the id of
      - Add `.json` onto the end of the url & reload
      - Copy the text and paste it into a [JSON formatter](https://jsonformatter.curiousconcept.com)
      - Find the value for `idList` in the parsed json
-5. Create a **docker-compose.yml** using your auth and the ids of those lists
+5. Enable [Custom Fields](https://trello.com/b/ssZMnPTq/projectscms/power-up/56d5e249a98895a9797bebb9) under Power-Ups for board *
```

Comment:
I'd rather have custom fields as optional so everyone who uses husky doesn't have to have that Power-Up (seen as Trello only lets you have one).

If the `DATE_ID` was namespaced as `TIMELINE_DATE_ID` then it could be opt-in when you use the timeline plugin, then I guess this section would go into a section about setting up the timeline, rather than the initial setup?

Reply:
Sorted